packages/video_player/video_player_web/lib/video_player_web.dart

```diff
@@ -170,4 +170,22 @@ class VideoPlayerPlugin extends VideoPlayerPlatform {
   /// Sets the audio mode to mix with other sources (ignored).
   @override
   Future<void> setMixWithOthers(bool mixWithOthers) => Future<void>.value();
+
+  @override
+  Future<List<VideoAudioTrack>> getAudioTracks(int playerId) async {
+    // Web platform does not support audio track selection
```

Comment:
Is this a limitation of web, or just due to it not being implemented?

Reply:
haven't explored the web apis for this. for now, i mean that its not supported for web yet. will update the comment